static/blfields.js

```diff
@@ -19,10 +19,10 @@ var blFields = [
 "placeOfReceipt",
 "placeOfDelivery",
 "onwardInlandRoutingFinalDestination",
-"containerNumbers",
+//"containerNumbers",
 "sizeTypes",
 "sealNos",
-"descriptionOfGoods",
+//"descriptionOfGoods",
```

Comment:
Why remove descriptionOfGoods and containerNumbers? They need to appear on the generated pdf. "Description of Goods" should be a field, not a title. Feel free to add the title "Particulars declared by Shipper but not acknowledged by the Carrier" (in accordance with the BL pdf template)